Source/Core/Core/IOS/ES/ES.h

```diff
@@ -92,8 +78,8 @@ class ES final : public Device
 
     u16 gid = 0;
     u32 uid = 0;
+    // The same context is used for both title imports and exports.
     TitleImportContext title_import;
```

Comment:
I changed `title_import` to `title_import_export` in https://github.com/dolphin-emu/dolphin/pull/5733/commits/1ba1b51606b2ed7f2c65da8733f59ab86b00561d, and it actually doesn't look too bad (and it makes it really clear that the same context is used for both operations). What do you think?